homeassistant/components/light/tradfri.py

```diff
@@ -184,7 +185,10 @@ def device_state_attributes(self):
         """Return the devices' state attributes."""
         info = self._light.device_info
 
-        attrs = {}
+        attrs = {
+            'manufacturer': info.manufacturer,
```

Comment:
This type of info should not go into state attributes according to @balloob.

Reply:
We change the opinion. It was complexity VS simplify and at the end we have no big benefit if we handle that in a separate pool and lost the simple core handling. Also if we look into future and AI...

Reply:
Ok, I didn't see that discussion.

Reply:
It wasn't done here, what I can tell:
https://github.com/home-assistant/home-assistant/issues/10732

Reply:
Correct was a discord discuss :+1:

Reply:
Maybe you can copy the bottom line and new direction from your discussion with @balloob to that issue, so that everyone that are following our development can take note?

It's important that we share information regarding things like this.

Reply:
No no, I still think that those shouldn't be in the state machine. We should have a new place if we care about this info.

Reply:
Ok